build.gradle

```diff
@@ -84,6 +90,7 @@ class Config {
 	public String liferayDockerImageId = ""
 	public String namespace = "lrswde"
 	public List<String> services = new ArrayList<String>()
+	public String webserverHostnames = "my.example.com"
```

Comment:
Do we actually want `my.example.com` to be the default value here? It seems error-prone to me, since someone might forget to change it.